tests/fixtures/core.py

```diff
@@ -471,7 +471,7 @@ def project(test_dir, project_init_service):
     # clean-up
     Project.deactivate()
     os.chdir(test_dir)
-    shutil.rmtree(project.root)
+    # shutil.rmtree(project.root)
```

Comment:
Should we do a full cleanup of all the pytest temp directories at the end of testing instead?

Also should we remove all the comments?